src/Containers.jsx

```diff
@@ -40,19 +42,84 @@ class Containers extends React.Component {
         }));
     }
 
-    //TODO
-    stopContainer(container) {
-        return undefined;
+    stopContainer(container, timeout) {
+        document.body.classList.add('busy-cursor');
+        if (!timeout) {
+            timeout = 10;
+        }
+        utils.varlinkCall(utils.PODMAN, "io.podman.StopContainer", {name: container.ID, timeout: timeout})
+            .then(reply => {
+                if (!reply.container) {
+                    return;
+                }
+                utils.updateContainers()
+                    .then((reply) => {
+                        // console.log(reply.newContainers);
+                        this.props.updateContainers(reply.newContainers);
+                        document.body.classList.remove('busy-cursor');
+                    })
+                    .catch(ex => {
+                        console.error("Failed to do Update Container:", JSON.stringify(ex))
+                        document.body.classList.remove('busy-cursor');
+                    })
+            })
```

Comment:
This code gets repeated several times. It seems more elegant if you let the top-level `app.jsx` component's `updateContainers()` do the `then` and `catch` handling, and then each subcomponent just has to do

    utils.varlinkCall(...).then(this.props.updateContainers)

There's going to be a lot more of these in the future, and at some point this is hopefully being replaced with a proper notification mechanism. So this should be centralized as much as possible.